docs/source/release_notes/release-notes.rst

```diff
@@ -42,6 +42,12 @@ develop
     *    - Type
          - Change
 
+    *    - |fixed|
```

Comment:
The release notes are basically irrelevant at this branch - I don't think they get published anywhere.